cloudferrylib/os/compute/nova_compute.py

```diff
@@ -428,20 +428,56 @@ def _deploy_instances(self, info_compute):
     def create_instance(self, **kwargs):
         return self.nova_client.servers.create(**kwargs).id
 
-    def get_instances_list(self, detailed=True, search_opts=None,
-                           marker=None,
-                           limit=None):
+    def get_instances_list(self, tenant_id=None, network_name=None, **kwargs):
```

Comment:
Is there any good reason for not passing `tenant_id` and `network_name` inside `search_opts`?